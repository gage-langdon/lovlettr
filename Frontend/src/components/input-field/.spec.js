import React from 'react';
import { shallow } from 'enzyme';
//component
import Input from './index';

describe('Input field component', () => {
  it('Will render with no props', () => {
    const wrap = shallow(<Input />);
    expect(wrap).toMatchSnapshot();
  });

  it('Will accept all props, will be hidden', () => {
    const wrap = shallow(
      <Input
        onChange={() => console.log('hit')}
        value={'hit'}
        disabled={true}
        placeholder={'not hit'}
        className="d-inline-flex flex-column"
      />
    );
  });
});
