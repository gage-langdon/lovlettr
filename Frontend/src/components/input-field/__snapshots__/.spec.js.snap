// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input field component Will render with no props 1`] = `
<input
  className=""
  disabled={false}
  maxLength={40}
  onChange={[Function]}
  placeholder=""
  type="text"
  value=""
/>
`;
